Console/Command/NostoClearQueueCommand.php

```diff
@@ -0,0 +1,201 @@
+
+    public function __construct(
+        ResourceConnection $resourceConnection,
+        Config $amqpConfig
+    ) {
+        $this->resourceConnection = $resourceConnection;
+        $this->amqpConfig = $amqpConfig;
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        // Define command name.
+        $this->setName('nosto:clear:queue')
+            ->setDescription('Clear all message queues for Nosto product sync topics.');
+        parent::configure();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        try {
+            foreach (self::QUEUE_TOPICS as $topicName) {
+                $this->clearQueue($topicName, $io);
+            }
+
+            $io->success('Successfully cleared message queues.');
+            return 0;
+        } catch (NostoException $e) {
+            $io->error('An error occurred while clearing message queues: ' . $e->getMessage());
+            return 1;
+        }
+    }
+
+    /**
+     * Clear MySql and RabbitMq queues by name.
+     *
+     * @param string $topicName
+     * @param SymfonyStyle $io
+     * @return void
+     */
+    private function clearQueue(string $topicName, SymfonyStyle $io): void
+    {
+        $this->clearRabbitMQQueue($topicName, $io);
+        $this->clearDBQueues($topicName, $io);
+    }
+
+    /**
+     * Clear DB.
+     *
+     * @param string $topicName
+     * @param SymfonyStyle $io
+     * @return void
+     */
+    private function clearDBQueues(string $topicName, SymfonyStyle $io): void
+    {
+        // Get connection.
+        $connection = $this->resourceConnection->getConnection();
+
+        // Start DB transaction.
+        $connection->beginTransaction();
+        try {
+            // Emptying DB tables.
+            $this->clearQueueMessages($topicName, $connection);
+            $this->clearRelatedRecords($topicName, $connection);
+            $connection->commit();
+        } catch (Exception $exception) {
+            $connection->rollBack();
+            $io->error('An error occurred while clearing DB queues for topic '
+                . $topicName . ': '
+                . $exception->getMessage()
+            );
+        }
+    }
+
+    /**
+     * Emptying queue message tables.
+     *
+     * @param string $topicName
+     * @param $connection
+     * @return void
+     */
+    private function clearQueueMessages(string $topicName, $connection): void
+    {
+        $queueMessageTable = $this->resourceConnection->getTableName('queue_message');
+        $queueMessageStatusTable = $this->resourceConnection->getTableName('queue_message_status');
+
+        // Get all IDs from "queue_message" table.
+        $select = $connection->select()
+            ->from($queueMessageTable, ['id'])
+            ->where('topic_name = ?', $topicName);
+        $messageIds = $connection->fetchCol($select);
```

Comment:
If you wanna go with this approach, you might wanna paginate this. Suppose you have 1 billion messages, wouldn't be nice to load all of them in memory at once.